src/script/cpp_api/s_env.cpp

```diff
@@ -177,21 +280,9 @@ void ScriptApiEnv::initializeEnvironment(ServerEnvironment *env)
 		int id = lua_tonumber(L, -2);
 		int current_lbm = lua_gettop(L);
 
-		std::set<std::string> trigger_contents;
+		std::vector<std::string> trigger_contents;
```

Comment:
Looks like new code is not removing duplicates.

Reply:
correct, does it matter?